SportsHub/UnitTests/Controllers/TestUserController.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System.Net;
+using System.Security.Claims;
+using AutoMapper;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Moq;
+using SportsHub.Api.Controllers;
+using SportsHub.Api.DTOs;
+using SportsHub.AppService.Services;
+using SportsHub.Domain.Models;
+using UnitTests.MockData;
+using Xunit;
+
+namespace UnitTests.Controllers;
+
+public class TestUserController
+{
+    private readonly Mock<IUserService> _userService;
+    private readonly UserController _userController;
+    private readonly IMapper _mapper;
+
+    public TestUserController()
+    {
+        if (_mapper == null)
+        {
+            var mappingConfig = new MapperConfiguration(x => x.CreateMap<User, UserResponseDTO>());
```

Comment:
Do a file with a mapping configuration exists in the project?

Reply:
No . I am doing the configuration for the mapper in the constructor

Reply:
I found `UserMapping` in `Api` and it contains mapping configuration. You can use it here. No need to create a new mapping.